### ISSUE TITLE
Free proxies :)

### ISSUE THREAD

**Issue by Lukkez** [davidmann4/ogar-feeder-bot#426]
Some proxy list i will put here (to people don't ask much proxies or don't ask):

4-19-2016 Socks5

100.7.34.243:26966
103.195.3.126:8080
108.192.17.43:42556
110.172.179.84:29081
117.74.120.136:8080
120.151.24.16:16702
137.118.142.239:10200
143.255.71.201:29081
150.145.1.87:9050
151.80.66.159:8975
166.62.82.206:18628
166.62.97.243:18628
168.167.88.85:3124
173.80.105.160:34320
176.107.198.63:19828
177.69.247.32:29081
177.75.116.127:29081
178.119.88.156:34438
178.150.100.78:24009
178.150.67.75:61917
178.151.244.110:6767
179.108.246.8:43011
180.153.87.22:10080
187.1.37.29:29081
187.160.128.204:15212
187.160.144.253:11101
187.160.146.136:10536
187.160.147.19:10419
187.160.147.91:10491
187.160.157.108:19932
187.160.157.217:19849
187.160.181.149:13637
187.160.205.162:30210
187.160.212.245:28501
187.160.49.199:35431
187.160.67.129:63521
187.160.82.76:59884
187.160.97.64:56032
187.161.104.178:54035
187.161.105.126:53983
187.161.117.189:52764
187.161.154.156:18509
187.161.155.241:18272
187.161.17.55:43670
187.161.20.89:45048
187.161.217.31:25278
187.161.22.77:44524
187.161.226.109:22988
187.161.23.173:44044
187.19.175.97:29081
187.241.174.208:10200
187.241.44.109:10200
188.40.128.240:8765
189.214.156.143:18537
189.218.106.176:10000
189.218.106.97:55227
189.218.126.220:49926
189.218.169.169:15235
189.218.200.226:30008
189.218.201.241:29739
189.218.240.214:19724
189.218.43.81:38539
189.218.60.91:33153
189.218.63.147:33353
189.218.63.234:33328
189.218.97.87:56461
189.219.101.6:55517
189.219.112.125:52646
189.219.129.190:15461
189.219.136.207:13588
189.219.136.29:13766
189.219.138.148:14159
189.219.190.80:10907
189.219.199.105:31410
189.219.199.247:32225
189.219.49.198:35869
189.219.55.164:10000
189.219.85.223:59396
189.219.85.80:59531
189.219.89.191:58468
189.219.90.254:59173
189.219.96.226:56633
190.68.66.36:38093
192.99.152.125:8975
193.107.113.28:59056
193.182.181.79:10200
197.220.206.106:29081
199.241.229.229:14577
199.47.207.192:10200
201.166.208.127:16617
201.166.209.82:16388
201.166.216.209:14471
201.172.101.186:44054
201.172.17.55:55451
201.172.71.23:36539
201.172.84.197:40297

**Comment by ghost** [comment]
@Lukkez  can I make my own proxy with c9.io :)

**Comment by hassanila** [comment]
No.... bro.. you want fresh socks? Contact me on skype Ive got method.   Tmrow.

**Comment by DeadWarrior58** [comment]
@hassanila97 what's your skype?

**Comment by hassanila** [comment]
:D...  sry but I know this guy.. thats why I will give him this... I cant tell every1 cuz If we all use same premium proxies... then RIP BOTS

**Comment by ElieeCh** [comment]
@DeadWarrior58 If you want proxies just use \* http://vip-socks24.blogspot.de/ \*  or \* http://proxylist.hidemyass.com/ *
Honestly, they both work pretty fine and you don't need better ones for agario... 
If you want to know if they work you can past them here \* http://www.checker.freeproxy.ru/checker/  \* and than copy the ones that are working.

**Comment by ghost** [comment]
@hassanila97  I have good proxies but I want make my own
